feature-libs/checkout/base/components/checkout-review/checkout-review-payment/checkout-review-payment.component.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { PaymentDetails } from '@spartacus/cart/base/root';
+import {
+  CheckoutPaymentFacade,
+  CheckoutStepType,
+} from '@spartacus/checkout/base/root';
+import { TranslationService } from '@spartacus/core';
+import { Card, ICON_TYPE } from '@spartacus/storefront';
+import { combineLatest, Observable } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+import { CheckoutStepService } from '../../services/checkout-step.service';
+
+@Component({
+  selector: 'cx-checkout-review-payment',
+  templateUrl: './checkout-review-payment.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CheckoutReviewPaymentComponent {
+  constructor(
+    protected checkoutStepService: CheckoutStepService,
+    protected checkoutPaymentFacade: CheckoutPaymentFacade,
+    protected translationService: TranslationService
+  ) {}
+
+  iconTypes = ICON_TYPE;
+
+  checkoutStepTypePaymentDetails = CheckoutStepType.PAYMENT_DETAILS;
```

Comment:
how about `paymentDetailsStepRoute = this.checkoutStepService.getCheckoutStepRoute(CheckoutStepType.PAYMENT_DETAILS);`, and in html, use `paymentDetailsStepRoute` directly.

Reply:
same thing in other review components.